config.example.wikimedia.yaml

```diff
@@ -47,11 +47,7 @@ default_project: &default_project
         pdf:
           # Cache PDF for 5 minutes since it's not purged
           cache_control: s-maxage=600, max-age=600
-          uri: https://pdfrender-beta.wmflabs.org
           new_uri: https://proton-beta.wmflabs.org
```

Comment:
maybe let's rename new_uri to uri?

Reply:
I was thinking about it, but to keep it backwards compatible, I opted with leaving it as-is. Perhaps we could go with this now and revisit in the future?